presto-main/src/test/java/com/facebook/presto/operator/TestGroupByHash.java

```diff
@@ -282,8 +282,8 @@ else if (type == BIGINT) {
                 });
         groupByHash.addPage(new Page(valuesBlock, hashBlock)).process();
 
-        // assert we call update memory every time we rehash; the rehash count = log2(length / FILL_RATIO)
-        assertEquals(rehashCount.get(), log2(length / 0.75, RoundingMode.FLOOR));
+        // assert we call update memory twice every time we rehash; the rehash count = log2(length / FILL_RATIO)
```

Comment:
update the comment for `the rehash count = 2 * log2(length / FILL_RATION)`

Reply:
rebased and updated the comment!

thanks for the review.